source/_posts/2024-09-04-release-20249.markdown

```diff
@@ -278,145 +248,269 @@ it is inevitable.
+
+[@ALERTua]: https://github.com/ALERTua
+[#124619]: https://github.com/home-assistant/core/pull/124619
+
+{% enddetails %}
+
+{% details "Proximity" %}
+
+In [2024.2 we introduced new sensor entities](/blog/2024/02/07/release-20242/#revamped-proximity-integration)
+to the [Proximity](/integrations/proximity) {% term integration %} and therefore deprecated the `proximity` entity.
+
+Now this deprecated `proximity` entity has been removed.
+
+([@mib1185] - [#123158]) ([documentation](/integrations/proximity))
+
+[@mib1185]: https://github.com/mib1185
+[#123158]: https://github.com/home-assistant/core/pull/123158
 
 {% enddetails %}
 
 {% details "Sensibo" %}
 
-Fix Pure AQI value sensor in Sensibo ([@gjohansson-ST] - [#124151]) ([sensibo docs]) (breaking-change)
+The previous PM2.5 sensor for Pure devices was incorrectly reporting a PM2.5 value, but it's actually a number representing an AQI level. The sensor will now provide the new states of 'good', 'moderate' and 'bad'.
+
+[@gjohansson-ST] - [#124151] ([documentation](/integrations/sensibo))
 
 [#124151]: https://github.com/home-assistant/core/pull/124151
 
 {% enddetails %}
 
-{% details "Overkiz" %}
+{% details "Starline" %}
 
-Update overkiz Atlantic Water Heater operation mode switching ([@ALERTua] - [#124619]) ([overkiz docs]) (breaking-change)
+The horn switch has been deprecated since 2024.1 and is now removed.
 
-[#124619]: https://github.com/home-assistant/core/pull/124619
+Instead of switch a horn button entity was added. Please update your automations accordingly.
+
+([@gjohansson-ST] - [#123608]) ([documentation](/integrations/starline))
+
+[#123608]: https://github.com/home-assistant/core/pull/123608
 
 {% enddetails %}
 
-{% details "Sisyphus" %}
+{% details "System monitor" %}
 
-Disable sisyphus integration ([@bdraco] - [#124742]) ([sisyphus docs]) (breaking-change)
+Removes the deprecated process sensor from the System Monitor integration.
+This sensor has been provided as a binary sensor for 6 months raising a repair issue.
 
-[#124742]: https://github.com/home-assistant/core/pull/124742
+[@gjohansson-ST] - [#123616] ([documentation](/integrations/systemmonitor))
+
+[#123616]: https://github.com/home-assistant/core/pull/123616
 
 {% enddetails %}
 
-{% details "MQTT" %}
+{% details "TP-Link Smart Home" %}
 
-Cleanup removed `schema` option from mqtt vacuum platform ([@jbouwh] - [#124722]) ([mqtt docs]) (breaking-change)
+The binary sensor for "Update available" will no longer be provided by the TP-Link Smart Home integration.
 
-[#124722]: https://github.com/home-assistant/core/pull/124722
+It will be replaced in a subsequent release with the Update entity but is disabled for now due to stability issues when calling the TP Link cloud API to check the latest firmware version. This may cause devices to go offline and power cycled to function again. The sensor was also unreliable and may have returned incorrect data because of rate limiting from the TP Link cloud.
```

Comment:
```suggestion
It will be replaced in a subsequent release with the Update entity but is disabled for now due to stability issues when calling the TP-Link cloud API to check the latest firmware version. This may cause devices to go offline. They need to be power-cycled to function again. The sensor was also unreliable and may have returned incorrect data because of rate limiting from the TP-Link cloud.
```